src/Governance.ts

```diff
@@ -80,4 +82,59 @@ export class Governance {
 
     return balanceToBigNumber(minimumProposalDeposit);
   }
+
+  /**
+   * Get how long in blocks the proposal cool off period and proposal ballot are valid
```

Comment:
Retrieve the proposal time frames. This includes:
- Amount of blocks from proposal creation until the proposal can be voted on (cool off)
- Amount of blocks from when cool off ends until the voting period is over (duration)